Sources/Fuzzilli/Core/Corpus.swift

```diff
@@ -36,7 +36,7 @@ public class Corpus: ComponentBase {
     private let minMutationsPerSample: Int
     
     /// The current set of interesting programs used for mutations.
-    private var programs: RingBuffer<Program>
+    public var programs: RingBuffer<Program>
```

Comment:
This probably shouldn't be public... Maybe we can make Corpus conform to [Collection](https://developer.apple.com/documentation/swift/collection) so that you can just do `for program in corpus`? For that, probably Corpus can just forward to the RingBuffer methods, since the RingBuffer is a Collection.